.github/workflows/TagBot.yml

```diff
@@ -9,3 +9,4 @@ jobs:
       - uses: JuliaRegistries/TagBot@v1
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
+          ssh: ${{ secrets.DOCUMENTER_KEY }}
```

Comment:
According to the Documenter docs this is needed to build the documentation for new tags